### ISSUE TITLE
ERROR: No matching distribution found for aicspylibczi

### ISSUE THREAD

**Issue by hd016** [AllenCellModeling/aicspylibczi#28]
**My conda env:**
```
          shell level : 2
          conda version : 4.5.11
    conda-build version : 3.0.22
         python version : 3.6.2.final.0
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/osx-64
                          https://repo.anaconda.com/pkgs/pro/noarch
               platform : osx-64
             user-agent : conda/4.5.11 requests/2.18.4 CPython/3.6.2 Darwin/15.6.0 OSX/10.11.6
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

```
Python 3.6.8
pip 19.3.1 
```

```
ERROR: Could not find a version that satisfies the requirement aicspylibczi (from versions: none)
ERROR: No matching distribution found for aicspylibczi
```

**Comment by heeler** [comment]
Addressed by release 2.5 - wheels for 3.6, 3.7, and 3.8 - should also support OSX10.13 through latest

This should expand support for python 3.6.

**Comment by bryantChhun** [comment]
I'm receiving this error on any 2.7 release while installing as a dependency (or individually) from aicsimageio

```
ERROR: Could not find a version that satisfies the requirement aicspylibczi~=2.7.5 (from aicsimageio)
ERROR: No matching distribution found for aicspylibczi~=2.7.5
```

similar env as the user above:

```
            shell level : 1
       user config file : /Users/bryant.chhun/.condarc
 populated config files : /Users/bryant.chhun/.condarc
          conda version : 4.8.5
    conda-build version : 3.10.5
         python version : 3.6.11.final.0
       virtual packages : __osx=10.13.6
       base environment : /Users/bryant.chhun/anaconda3  (writable)
           channel URLs : https://conda.anaconda.org/talley/osx-64
                          https://conda.anaconda.org/talley/noarch
                          https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
               platform : osx-64
             user-agent : conda/4.8.5 requests/2.22.0 CPython/3.6.11 Darwin/17.7.0 OSX/10.13.6
                UID:GID : 502:20
             netrc file : None
           offline mode : False
```

```
python 3.7.9
pip 20.3.1
```
